main.tf

```diff
@@ -5,29 +5,56 @@ terraform {
     region  = "us-west-2"
     key     = "indent/terraform.tfstate"
   }
+}
 
+# Configure the Okta Provider
+provider "okta" {
+  org_name  = var.okta_prefix
+  base_url  = "okta.com"
+  api_token = var.okta_token
 }
 
 # Indent + Okta Integration
 
-# Details: https://github.com/indentapis/integrations/tree/f494cef86094c3b40ac124e3159f5f3391c7e6c8/packages/stable/indent-integration-okta
-# Last Change: https://github.com/indentapis/integrations/commit/f494cef86094c3b40ac124e3159f5f3391c7e6c8
+# Details: https://github.com/indentapis/integrations/tree/f0cea0e363f8950c7a217d186df6c377ed52e9d7/packages/stable/indent-integration-okta
+# Last Change: https://github.com/indentapis/integrations/commit/f0cea0e363f8950c7a217d186df6c377ed52e9d7
 
 module "idt-okta-webhook" {
   source                = "git::https://github.com/indentapis/integrations//terraform/modules/indent_runtime_aws_lambda"
   name                  = "idt-okta-webhook"
   indent_webhook_secret = var.indent_webhook_secret
   artifact = {
     bucket       = "indent-artifacts-us-west-2"
-    function_key = "webhooks/aws/lambda/okta-f494cef86094c3b40ac124e3159f5f3391c7e6c8-function.zip"
-    deps_key     = "webhooks/aws/lambda/okta-f494cef86094c3b40ac124e3159f5f3391c7e6c8-deps.zip"
+    function_key = "webhooks/aws/lambda/okta-f0cea0e363f8950c7a217d186df6c377ed52e9d7-function.zip"
+    deps_key     = "webhooks/aws/lambda/okta-f0cea0e363f8950c7a217d186df6c377ed52e9d7-deps.zip"
   }
   env = {
-    OKTA_DOMAIN       = var.okta_domain
-    OKTA_TOKEN        = var.okta_token
+    OKTA_DOMAIN       = "${var.okta_prefix}.okta.com"
+    # OKTA_TOKEN        = var.okta_token
     OKTA_SLACK_APP_ID = var.okta_slack_app_id
-    OKTA_CLIENT_ID    = var.okta_client_id
-    OKTA_PRIVATE_KEY  = var.okta_private_key
+    OKTA_CLIENT_ID    = okta_app_oauth.indent.id
+    OKTA_PRIVATE_KEY  = file("./private.pem")
```

Comment:
this is generated by the jwk.rb script, which also sets a TF_VAR for the jwk `n` value. Clean this up -- make them both vars, probably (and ignore changes as well).